File: prime or not.py
# prime number or not from 1-100
minimum=1
maximum=100
print("prime number between",minimum,"and",maximum,"are:")
for num in range(minimum,maximum+1):
    if num>1:
        for i in range(2,num):
            if(num % i)==0:
                break
        else:
            print(num)

File: prime or not.cpp
#include<iostream>
using namespace std;
int main()
{
    int num, i, a=0;
    cout<<"Enter a Number: ";
    cin>>num;
    for(i=2; i<num; i++)
    {
        if(num%i==0)
        {
            a++;
            break;
        }
    }
    if(a==0)
        cout<<"\nIt is a Prime Number";
    else
        cout<<"\nIt is not a Prime Number";
    cout<<endl;
    return 0;
}